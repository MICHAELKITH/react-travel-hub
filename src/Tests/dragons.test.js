import { Provider } from 'react-redux';
import TestRenderer from 'react-test-renderer';
import Dragons from '../components/dragon';
import Store from '../slices/store';

describe('Missions', () => {
  test('Render Dragons Component', () => {
    const tree = TestRenderer.create(
      <Provider store={Store}>
        <Dragons />
      </Provider>,
    ).toJSON();
    expect(tree).toMatchSnapshot();
  });
});
