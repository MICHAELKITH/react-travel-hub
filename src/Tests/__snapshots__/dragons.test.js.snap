// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Missions Render Dragons Component 1`] = `<div />`;
